opm/output/data/Wells.hpp

```diff
@@ -101,6 +103,8 @@ namespace Opm {
             template <class MessageBufferType>
             void read(MessageBufferType& buffer);
 
+            template <class T> friend struct Mpi::Packing;
```

Comment:
So this will, in principle, mean that `Mpi::Packing<MyEvilType>` has read/write access to the private data members of `data::Rates`.  It probably doesn't matter, but is something we should keep in mind.  Is there a way to limit **which** specializations of `Mpi::Packing<>` are granted special access privileges?  Maybe something along the lines of
```C++
friend struct Mpi::Packing<Rates>
```

Reply:
I will check if that works.

Reply:
It works, and I have changed it.